### ISSUE TITLE
audio playing after pause() was called on iOS

### ISSUE THREAD

**Issue by MiroLiebschner** [bluefireteam/audioplayers#1034]
As I want to start the audio at a certain position I have to `play(url, position: position)`. When calling `pause()` afterwards the audio does not always stop on iOS!  

**Full Description**
In the `_initAudioPlayer()` I call `play(url, position: position)` to skip to the position I want the audio to start at. As the user needs to play the audio manually I need to stop the audio immediately with `_pause()`. This might or might not pause the audio.
I can't call `seek()` as this only works after `play()` was called first!

**Code to Reproduce**
in `_initAudioPlayer()` I call this function: 
'''	void jumpToStartPosition() async {
		if (widget.startPositionInMilliseconds == 0) return;
		await _play(
				position: Duration(milliseconds: widget.startPositionInMilliseconds));
		Timer(Duration(milliseconds: 80), () {
			_pause();
		});
	}'''

The problem only occurs on iOS. Tested on iOS simulator. 
Flutter (Channel stable, 2.5.3, on macOS 11.5.2

**Comment by tsunhousam91** [comment]
I also have this problem, I want to call pause immediately after play() is called, but it will failed on iOS, like @MiroLiebschner , I delay a moment for pause() being called, but still face the problem the pause might failed sometimes.

**Comment by MiroLiebschner** [comment]
Yeah right now I do not see a option to skip to a position before playing the audio && I do not see an option to reliable stop the audio right after the skip was done. So in other words it is not possible to start at a certain position.

**Comment by lorydi3001** [comment]
Same problem. And it keeps playing in background for a while which is not acceptable for games.
